From: Alexander Mikhaylenko <alexm@gnome.org>
Date: Tue, 13 Oct 2020 18:54:49 +0500
Subject: Add padding for HdyViewSwitcherTitle

Make sure it moves to the bottom on narrow sizes.
---
 gtk/theme/Adwaita/_common.scss                   | 5 +++++
 gtk/theme/Adwaita/gtk-contained-dark.css         | 2 ++
 gtk/theme/Adwaita/gtk-contained.css              | 2 ++
 gtk/theme/HighContrast/gtk-contained-inverse.css | 2 ++
 gtk/theme/HighContrast/gtk-contained.css         | 2 ++
 5 files changed, 13 insertions(+)

diff --git a/gtk/theme/Adwaita/_common.scss b/gtk/theme/Adwaita/_common.scss
index 19b6f38..e40e429 100644
--- a/gtk/theme/Adwaita/_common.scss
+++ b/gtk/theme/Adwaita/_common.scss
@@ -4868,6 +4868,11 @@ viewswitcher {
     padding: 0;
   }
 
+  headerbar & {
+    padding-left: 70px;
+    padding-right: 70px;
+  }
+
   button {
     border-radius: 0;
     border-top: 0;
diff --git a/gtk/theme/Adwaita/gtk-contained-dark.css b/gtk/theme/Adwaita/gtk-contained-dark.css
index 9797b3a..3cd917d 100644
--- a/gtk/theme/Adwaita/gtk-contained-dark.css
+++ b/gtk/theme/Adwaita/gtk-contained-dark.css
@@ -1958,6 +1958,8 @@ popover.emoji-completion .emoji:hover { background: #424242; }
 /************ libhandy * */
 viewswitcher, viewswitcher button { margin: 0; padding: 0; }
 
+headerbar viewswitcher { padding-left: 70px; padding-right: 70px; }
+
 viewswitcher button { border-radius: 0; border-top: 0; border-bottom: 0; box-shadow: none; font-size: 1rem; }
 
 viewswitcher button:not(:checked):not(:hover) { background: transparent; }
diff --git a/gtk/theme/Adwaita/gtk-contained.css b/gtk/theme/Adwaita/gtk-contained.css
index 72946ec..4eff74f 100644
--- a/gtk/theme/Adwaita/gtk-contained.css
+++ b/gtk/theme/Adwaita/gtk-contained.css
@@ -1974,6 +1974,8 @@ popover.emoji-completion .emoji:hover { background: white; }
 /************ libhandy * */
 viewswitcher, viewswitcher button { margin: 0; padding: 0; }
 
+headerbar viewswitcher { padding-left: 70px; padding-right: 70px; }
+
 viewswitcher button { border-radius: 0; border-top: 0; border-bottom: 0; box-shadow: none; font-size: 1rem; }
 
 viewswitcher button:not(:checked):not(:hover) { background: transparent; }
diff --git a/gtk/theme/HighContrast/gtk-contained-inverse.css b/gtk/theme/HighContrast/gtk-contained-inverse.css
index eb633dc..5a2cf09 100644
--- a/gtk/theme/HighContrast/gtk-contained-inverse.css
+++ b/gtk/theme/HighContrast/gtk-contained-inverse.css
@@ -2038,6 +2038,8 @@ popover.emoji-completion .emoji:hover { background: #424242; }
 /************ libhandy * */
 viewswitcher, viewswitcher button { margin: 0; padding: 0; }
 
+headerbar viewswitcher { padding-left: 70px; padding-right: 70px; }
+
 viewswitcher button { border-radius: 0; border-top: 0; border-bottom: 0; box-shadow: none; font-size: 1rem; }
 
 viewswitcher button:not(:checked):not(:hover) { background: transparent; }
diff --git a/gtk/theme/HighContrast/gtk-contained.css b/gtk/theme/HighContrast/gtk-contained.css
index 95d09e5..ba598cc 100644
--- a/gtk/theme/HighContrast/gtk-contained.css
+++ b/gtk/theme/HighContrast/gtk-contained.css
@@ -2054,6 +2054,8 @@ popover.emoji-completion .emoji:hover { background: white; }
 /************ libhandy * */
 viewswitcher, viewswitcher button { margin: 0; padding: 0; }
 
+headerbar viewswitcher { padding-left: 70px; padding-right: 70px; }
+
 viewswitcher button { border-radius: 0; border-top: 0; border-bottom: 0; box-shadow: none; font-size: 1rem; }
 
 viewswitcher button:not(:checked):not(:hover) { background: transparent; }
